### PR TITLE
WIP: implement GMAOs choices

### PR DESCRIPTION
**Pull request by jswhit**
with some differences.
• assimilate amsua ch 14 without bias correction (iuse=4) - this is also different from NCEP ops
• for amsua, assimlate chs1-3 and 15
• for n15 amsua don't stop assimlating 1-5 and 15 (since these are still being used in NCEP ops)
• for n14 msu use ch 1

### CODE REVIEW

**Comment by jswhit** [comment]
@frolovsa I've implemented NASA's choices for all amsua, please take a look when you get a chance. We don't want to merge this into master, but keep it in a branch I guess - but I created a PR anyway to make it easier to review the changes.

File: satinfo/amsua_n19/readme [hunk 1]
```diff
@@ -1,3 +1,6 @@
-n19  20091215 000000  20091221 240000  amsua 14  1 2 3 4 5 6 7 8 9 10 11 12 13 14 15               # start date per Merra 
-n19  20091222 000000  20150114 060000  amsua 13  1 2 3 4 5 6 7   9 10 11 12 13 14 15               # channel 8 noisy per Merra
-n19  20150114 120000  21001231 240000  amsua 12  1 2 3 4 5 6     9 10 11 12 13 14 15               # turn off channel 7 as in operational satinfo
+# launched 20090209, dumps started 20009293, GSI started to use 20091215
```
Comment (frolovsa) [inline satinfo/amsua_n19/readme]: typo. "20009293" is not a valid date and i cant figure out what date you meant.

File: satinfo/msu_n14/readme [hunk 2]
```diff
@@ -1 +1,2 @@
-n14  19950119 000000  20061001 240000  msu    4  1 2 3 4	 # end date as in CFSR (Merra off 20061010)			
+# NOAA assimilated ch1, NASA did not
```
Comment (frolovsa) [inline satinfo/msu_n14/readme:1]: Jeff. where did the msu info come from? Should it be part of this PR?

File: satinfo/amsua_aqua/readme [hunk 3]
```diff
@@ -1,4 +1,8 @@
...
-aqua  20160923 180000  21001231 240000  amsua   0                                                            # chs1-2 detector failure, cannot QC
-# NCEP continues to assimilate 8-13
+# launched 20020504, dumps started 20040413 ('Centerfield-of-view'),
+# 20051018 ('Warmest/clearest field-of-view',i.e. airswm)
+# 20070221 ('Every field-of-view' dumps start)
+# chan 14 used with no bias correction (iuse=4)
+# chs 1-3,15 assimilated by NOAA, but not NASA
+aqua  20021012 000000  20071102 120000  amsua  14 1 2 3 4 5 6   8 9 10 11 12 13 14 15
+aqua  20071102 180000  20160923 120000  amsua   8           6   8 9 10 11 12 13 14    # ch4 noisy; QC affected ; turn off ch1-5,15
+aqua  20160923 180000  21001231 240000  amsua   0                                     # chs1-2 detector failure, turn off
```
Comment (frolovsa) [inline satinfo/amsua_aqua/readme:8]: Jeff. did you add a file that turns the satellite off? I am not seeing it in this PR. Maybe i am missing something.

File: satinfo/amsua_metop-c/readme [hunk 4]
```diff
@@ -1 +1 @@
-metop-c  20191107 060000  21001231 240000  amsua 14  1 2 3 4 5 6 7 8 9 10 11 12 13 14 15     # from operational satinfo
+metop-c  20191107 000000  21001231 240000  amsua 14  1 2 3 4 5 6 7 8 9 10 11 12 13 14 15
```
Comment (frolovsa) [inline satinfo/amsua_metop-c/readme]: Jeff the start date in notes is "2019**09**17" yours is "2019**11**07" I think you misread it. Or i misread it.

**Review by frolovsa — CHANGES_REQUESTED** [review]
Jeff. Looks good thank you. I found a couple of typos.

File: satinfo/msu_n14/readme (hunk 2 above)
Comment (jswhit) [reply satinfo/msu_n14/readme:1]: I added it since I had a little extra time

File: satinfo/amsua_aqua/readme (hunk 3 above)
Comment (jswhit) [reply satinfo/amsua_aqua/readme:8]: good catch - fixed now

File: satinfo/amsua_metop-c/readme (hunk 4 above)
Comment (jswhit) [reply satinfo/amsua_metop-c/readme]: fixed

File: satinfo/amsua_n19/readme (hunk 1 above)
Comment (jswhit) [reply satinfo/amsua_n19/readme]: should be 20090929 (fixed).  Info comes from historical/Satellite_Historical_Documentation.txt

**Comment by jswhit** [comment]
Thanks @frolovsa.  Since I want to keep this in a separate branch, should I just ping you next time I have an instrument updated?

**Comment by frolovsa** [comment]
yes @jswhit . If you want. You can make a PR into your GMAO branch. So all other PRs will be easy to read since they will be based-off GMAO and not master.

**Comment by jswhit** [comment]
good idea!

**Pull request closed without merging**